### ISSUE TITLE
UnpicklingError: invalid load key, 'H'

### ISSUE THREAD

**Issue by cnbeining** [bedapudi6788/deeppunct#7]
Hi,

I am using the `deeppunct_params_en` in `README.md` from https://drive.google.com/drive/folders/1Yd8cJaqfQkrJMbRVWIWtuyo4obTDYu-e

and got error

```
    178     if os.path.exists(params_path):
    179         print('Loading the params file')
--> 180         params = pickle.load(open(params_path, 'rb'))
    181         return params
    182 

UnpicklingError: invalid load key, 'H'.
```

Error can be reproduced on Python 3.5, 3.6 and 3.7. 

Are there any change introduced in the params file?

Thanks,

**Comment by bedapudi6788** [comment]
@cnbeining can you please reproduce the issue in a colab notebook if possible? That would help me in debugging the issue.

**Comment by zhushaojin** [comment]
Hi, I meet the same issue.

System environment: Python 3.7.3 and TensorFlow 1.14.0

The model was downloaded from your google driver "deeppunct_checkpoint_google_news"

I used the code 
`corrector = DeepCorrect('./deeppunct_checkpoint_google_news', 'checkpoint_path')`
Didn't sure what to fit in "'checkpoint_path'" so I just leaved it there.

An error occurred at "DeepCorrect.__init__()"
`DeepCorrect.deepcorrect_model = build_model("./deeppunct_checkpoint_google_news") `

The error report is the same
`_pickle.UnpicklingError: invalid load key, 'H'.`

Did I used the wrong model? 
Thanks
